Task5/css/header/__content.css

```diff
@@ -0,0 +1,69 @@
+.header__content {
+    position: absolute;
```

Comment:
I think it should be `relative` positioning but background-image should be absolute. Ideally, you should use `background-image` css property on `<header>` tag instead of this `<img>` with absolute positioning.